src/network.zig

```diff
@@ -972,20 +972,38 @@ pub const Protocols = struct {
 					game.Player.setPosBlocking(try reader.readVec(Vec3d));
 				},
 				.worldEditPos => {
-					const typ = try reader.readEnum(WorldEditPosition);
-					switch(typ) {
-						.selectedPos1, .selectedPos2 => {
-							const pos = try reader.readVec(Vec3i);
-							switch(typ) {
-								.selectedPos1 => game.Player.selectionPosition1 = pos,
-								.selectedPos2 => game.Player.selectionPosition2 = pos,
-								else => unreachable,
-							}
-						},
-						.clear => {
-							game.Player.selectionPosition1 = null;
-							game.Player.selectionPosition2 = null;
-						},
+					if(isServerSide(conn)) {
+						const typ = try reader.readEnum(WorldEditPosition);
+						switch(typ) {
+							.selectedPos1, .selectedPos2 => {
+								const pos = try reader.readVec(Vec3i);
```

Comment:
Instead of having this in both branches, and making nested switches, how about doing the following before the branch like this:
```zig
const pos: ?Vec3i = switch(typ) {
	.selectedPos1, .selectedPos2 => try reader.readVec(Vec3i),
	.clear => null,
};
```